server/controllers/shared/referralsController.test.ts

```diff
@@ -138,13 +141,18 @@ describe('ReferralsController', () => {
   })
 
   describe('sentenceInformation', () => {
+    const detailsSummaryListRows =
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      'details summary list rows' as any as Array<GovukFrontendSummaryListRowWithKeyAndValue>
```

Comment:
Using a mock here, rather than a string `[createMock< GovukFrontendSummaryListRowWithKeyAndValue>()]` might be a bit nicer type-wise.